lib/_tls_common.js

```diff
@@ -52,6 +54,13 @@ function SecureContext(secureProtocol, secureOptions, context) {
   if (secureOptions) this.context.setOptions(secureOptions);
 }
 
+function validateKeyCert(value, type) {
+  if (typeof value !== 'string' && !(value instanceof Buffer))
```

Comment:
I don’t think we can list `ArrayBuffer`, as those are not ArrayBufferViews themselves.

Maybe `['string', 'Buffer', 'TypedArray', 'DataView']`? That would match the message here: https://github.com/nodejs/node/blob/cde272a066923a1b93a10a8121b0ce572308d13c/lib/zlib.js#L231